src/plugins/LADTable/components/LADTable.vue

```diff
@@ -0,0 +1,94 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+ <template>
+    <table class="c-table c-lad-table">
+        <thead>
+            <tr>
+                <th>Name</th>
+                <th>Timestamp</th>
+                <th>Value</th>
+            </tr>
+        </thead>
+        <tbody>
+            <lad-row 
+                v-for="item in items"
+                :key="item.key"
+                :domainObject="item.domainObject">
+            </lad-row>
+        </tbody>
+    </table>
+ </template>
+
+ <script>
+ import LadRow from './LadRow.vue';
+
+ export default {
+    inject: ['openmct', 'domainObject'],
+    components: {
+        LadRow
+    },
+    data() {
+        return {
+            items: []
+        }
+    },
+    methods: {
+        indexOf(object, array) {
+            let index = -1;
+
+            array.forEach((o,i) => {
+                if(o.domainObject.identifier.key === object.key) {
```

Comment:
identifier cannot be compared by key alone.  

also, this would be a good case to use `_.findIndexBy`.